cmd/snap/cmd_snap_op.go

```diff
@@ -979,34 +1001,19 @@ func (x cmdSwitch) Execute(args []string) error {
 	name := string(x.Positional.Snap)
 	channel := string(x.Channel)
 
-	var msg string
+	switchCohort := x.Cohort != ""
+	switchChannel := x.Channel != ""
+
 	// some duplication between this and the two other switch-summarisers...
 	// in this one, we have three boolean things to check, meaning 2³=8 possibilities
 	// of which 3 are errors (which is why we look at this before running it)
```

Comment:
I think the comment needs adjusting to point out that the cases are split now into error ones (checked here) and non-error ones (checked in showDone)

Reply:
Done.